compose/backend.yml

```diff
@@ -2,6 +2,35 @@
 version: '3'
 
 services:
+
+  annotation-api:
+    image: hmctspublic.azurecr.io/em/anno:latest
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://annotation-api-db:5432/emannotationapp
+      - SPRING_DATASOURCE_USERNAME=emannotationapp
+      - SPRING_DATASOURCE_PASSWORD=emannotationapp
+      - S2S_BASE_URI=http://service-auth-provider-api:8080
+      - IDAM_API_BASE_URI=http://idam-api:8080
+      - ENABLE_LIQUIBASE=true
+      - S2S_NAMES_WHITELIST=sscs,divorce,ccd,em_gw,jui_webapp,pui_webapp,em_npa_app,ccd_gw,ccd_gateway
```

Comment:
Please remove the redundant elements for ccd. For ccd, we should have only 1 element - ccd_gw or ccd_gateway, couldn't recall now.

Reply:
Not sure the correct element, ccd_gw or ccd_gateway

Reply:
Ok, leave it to me pls.